policyengine_us/parameters/gov/states/dc/tax/income/credits/ptc/fraction_nonelderly.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   threshold_period: year
   threshold_unit: currency-USD  # federal AGI
   amount_unit: /1
+  uprating: gov.irs.uprating
```

Comment:
this won't currently work - it'll uprate both threshold and amount pending a core fix. see CA as an example for the interim

Reply:
got u, just changed the two threshold-related uprating